storage/boltdb/client_test.go

```diff
@@ -140,3 +149,155 @@ func BenchmarkSuiteLong(b *testing.B) {
 	}
 	testsuite.BenchmarkPathOperationsInLargeDb(b, longStore)
 }
+
+func BenchmarkClientWrite(b *testing.B) {
+	// setup db
+	ctx := testcontext.New(b)
+	defer ctx.Cleanup()
+	dbfile := ctx.File("testbolt.db")
+	dbs, err := NewShared(dbfile, "kbuckets", "nodes")
+	if err != nil {
+		fmt.Printf("failed to create db: %v\n", err)
```

Comment:
This (and all following `fmt.Printf()` calls in the benchmark functions) should be a `b.Fatalf()` call. The benchmark results are not valid if we have failed to create, close, or modify the DBs in the expected way. Plus if these are happening during automated testing, we really want to know about it.